en/docs/install-and-setup/setup/advance-configurations/configure-certificate-revocation.md

```diff
@@ -0,0 +1,43 @@
+# Verifying Certificate Revocation
+
+The default HTTPS transport listener (Secured Pass-Through) can verify with the certificate authority whether a certificate is still trusted before it completes an SSL connection. If the certificate authority has revoked the certificate, a connection will not be completed.
+
+When this feature is enabled, the transport listener verifies client
+certificates when a client tries to make an HTTPS connection with the server. Therefore, the client needs to send it's public certificate along with the requests to the server.
+
+When this feature is enabled, the server attempts to
+use the Online Certificate Status Protocol (OCSP) to verify with the
+certificate authority at the handshake phase of the SSL protocol. If the
+OCSP is not supported by the certificate authority, the server uses Certified Revocation Lists (CRL) instead. The verification
+process checks all the certificates in a certificate chain.
+
+To enable this feature for the HTTP Pass-Through, add the following parameters in the deployment.toml file. and set ```enable``` as ```true```.
```

Comment:
Let's provide the complete path to these configuration files. `deployment.toml` and `axis2.xml`